recipe/meta.yaml

```diff
@@ -31,6 +33,7 @@ test:
     - pkgconfig
     - pkgconfig.pkgconfig
   requires:
+    # - python <3.10
```

Comment:
This comment should be removed is it no longer necessary.

Reply:
the unnecessary comments were eliminated